### DISCUSSION TITLE
Unable to specify or represent an organisation part/department since ePO 4.0-rc.2

### DISCUSSION THREAD

**Discussion by schivmeister** [eprocurementontology/eprocurementontology#558] [Ideas]
# Issue

The property `epo:hasOrganisationUnit` that was present in ePO 3.1.0 [for mapping an organisation part name](https://github.com/OP-TED/ePO/issues/238) changed in 4.0-rc.1 to `epo:hasOrganisationUnitName`, and then in 4.0-rc.2, it was apparently removed, as evidenced by the [changelogs](https://github.com/OP-TED/ePO/compare/v4.0.0-rc.1...v4.0.0-rc.2):

```diff
- :hasOrganisationUnitName a owl:DatatypeProperty ;
-    rdfs:isDefinedBy :core ;
-    skos:definition """The name of a subpart of an organisation.
- Additional Information:
- E.g. the relevant department of a large organisation.
- We added this attribute in case of eProcurement notices where the Buyer is actually a part of an Organization and the rest of the properties apply to the rest of the Organisation as well. (org:Organization) """@en ;
-     skos:prefLabel "Has organisation unit name"@en .
```

However, the reason for this removal is not made clear in the commits or the logs.

This is required to map for e.g. the field `BT-16-Organization-TouchPoint` or `BT-16-Organization-Company` [in the eForms SDK](https://github.com/OP-TED/eForms-SDK/blob/1.10.1/fields/fields.json#L5893), which can identify a department in an organisation contact point or company address, with one of the following typical XML structures:

**Organisation Contact Point Address**
```xml
<Organization>
  <TouchPoint>
    <PostalAddress>
      <Department>
```

**Company Address**
```xml
<Company>
  <PostalAddress>
    <Department>
```

Since 4.0.0 ePO has introduced the notion of a _sub-organisation_ with the `org:subOrganizationOf` relationship (object property).

<img width="263" alt="image" src="https://github.com/OP-TED/ePO/assets/1151152/5c9279dc-6b74-4b69-9709-10c8837c300d">

If this was meant as a replacement, then it is not a very appropriate one to represent the semantics, i.e. that of only a line in the address specifying the department. We would not typically think of a department as a sub-organisation; it would not have the same basic characteristics of an organisation to qualify as a valid subsumption.

**Comment by andreea-pasare** [comment]
The decision to remove the attribute epo:hasOrganisationUnitName and add org:subOrganizationOf was done in order to align with [Core organization ontology](https://www.w3.org/ns/org) where the org:subOrganizationOf is defined as:
![image](https://github.com/OP-TED/ePO/assets/88081600/bfdd6293-2199-4548-956a-f5c5f0527bdd)

This modification is mentioned in the Change notes related to the release notes of ePO 4.0.0 that can be found [here](https://docs.ted.europa.eu/EPO/4.0/release-notes.html). 

As a note, a similar modelling approach is adopted in the latest release of [Core Business Vocabulary](https://semiceu.github.io/Core-Business-Vocabulary/releases/2.2.0/):
![image](https://github.com/OP-TED/ePO/assets/88081600/55aeaf4c-cd11-4c13-863a-101e3b03d03d)

**Comment by schivmeister** [comment]
@andreea-pasare thanks for the context, that's very useful to know. Unless I missed it again, the change notes you linked to does not have any information about this implied replacement -- only that at some point in the history one was added and the other removed. But it could also be a lack of intuition on my part.

I understand now that it was a conscious decision, and that at a fundamental level the department is a specialization of the organization. However, a sub-organization is very broad, and typically refers to things [like subsidiaries](https://schema.org/subOrganization). We see these notions discussed [here](http://mapping.semic.eu:81/vdm/description.vsp?namespace=cv&type=e7ebe945b8579640bed3c957d0656043&id=&format=), for example:

> The subOrganization and subOrganizationOf properties express the relationships between organizations in a hierarchical structure. **In contrast, hasUnit and unitOf are used to link to operational departments within an organization that may not generally exist in their own right.**

So it appears a further specialization will be helpful to have. I see such a specialization in [the Organization Ontology](https://www.w3.org/TR/vocab-org/) and perhaps the [Core Business Vocabulary](https://semiceu.github.io/Core-Business-Vocabulary/releases/2.2.0/) can follow suit:

```ttl
org:hasUnit a owl:ObjectProperty, rdf:Property;

    rdfs:label "has Unit"@en;
...
    rdfs:domain org:FormalOrganization;
    rdfs:range  org:OrganizationalUnit;
    rdfs:subPropertyOf org:hasSubOrganization;

    rdfs:comment """Indicates a unit which is part of this Organization, e.g. a Department within a larger FormalOrganization. Inverse of `org:unitOf`."""@en;
...
```
